### ISSUE TITLE
slider_input does not work if min != 0

### ISSUE THREAD

**Issue by rolfmblindgren** [Appsilon/shiny.semantic#362]
mwe:

```
library(semantic.dashboard)  # version 0.2.0
library(shiny.semantic) # 0.4.2

the_bug <- 0 # any other value, and the slider grays out.

ui <- semanticPage(
    
    segment(
        slider_input("slider_ex", 50, the_bug, 80, 2)
    )
)

server <- function(input, output) {}

shinyApp(ui, server)
```

If the value of the_bug is other than 0, the slider grays out and can't be used. This has caused a lot of my web apps to stop working.

**Comment by dokato** [comment]
Thank you @rolfmblindgren for reporting that. We'll try to fix that soon.

**Comment by ashbaldry** [comment]
Hi @rolfmblindgren, could you try the develop branch? I think this is fixed when resolving #351.

**Comment by dokato** [comment]
Great, this should be published on CRAN soon.
